tests/test-sources/plugins/by-name/perfanno/default.nix

```
@@ -0,0 +1,49 @@
+{
+  empty =
+    { pkgs, ... }:
+    {
+      plugins.perfanno.enable = !pkgs.stdenv.isDarwin;
+    };
+
+  example =
+    { pkgs, ... }:
+    {
+      plugins.perfanno = {
+        enable = !pkgs.stdenv.isDarwin;
+
+        settings = {
+          line_highlights.__raw = ''util.make_bg_highlights(nil, "#CC3300", 10)'';
+          vt_highlight.__raw = ''util.make_fg_highlight("#CC3300")'';
```

> example> ERROR: Error detected while processing /nix/store/v6ydz5x55nvr6ipplapbcwq4gil7kjy0-init.lua:
> example> E5113: Error while calling lua chunk: /nix/store/v6ydz5x55nvr6ipplapbcwq4gil7kjy0-init.lua:13: attempt to index global 'util' (a nil value)
> example> stack traceback:
> example>        /nix/store/v6ydz5x55nvr6ipplapbcwq4gil7kjy0-init.lua:13: in main chunk
> -- https://buildbot.nix-community.org/#/builders/4425/builds/442

Upstream has `local util = require("perfanno.util")` higher up in their example, so here we want:

```suggestion
          line_highlights.__raw = ''require("perfanno.util").make_bg_highlights(nil, "#CC3300", 10)'';
          vt_highlight.__raw = ''require("perfanno.util").make_fg_highlight("#CC3300")'';
```

---

Or alternatively, outside the test in the main plugin, we could add something like:

```nix
extraConfig = {
  # TODO: should we also `lib.mkBefore` or `lib.mkOrder` this definition?
  plugins.perfanno.luaConfig.pre = ''
    local util = require("perfanno.util")
  '';
};
```

This would add `util` to the scope for anyone adding code to the plugin's `luaConfig`, which could be helpful _or_ surprising 😅